crates/wasmcloud-host/src/manifest.rs

```diff
@@ -96,7 +96,7 @@ pub(crate) async fn generate_actor_start_messages(
             }
         } else {
             // load actor from OCI
-            if let Ok(a) = fetch_oci_bytes(&actor_ref, allow_latest)
+            if let Ok(a) = fetch_oci_bytes(&actor_ref, allow_latest, false)
```

Comment:
If you include a parameter for `allow_insecure`, the place where this function (`generate_actor_start_messages`) is called is from a context where the Host is `self`, so you'd have access to the Host `allow_insecure` option mentioned in the above comment